__tests__/app.test.js

```diff
@@ -75,6 +75,28 @@ describe("/api", () => {
             expect(articles).toBeSortedBy("created_at", { descending: true });
           });
       });
+      it("200: return status code 200, FILTER article data by specified topic", () => {
+        return supertest(app).get("/api/articles?topic=cats").expect(200)
+          .then((res) => {
+            const { articles } = res.body;
+            expect(articles.length).toBe(1)
+            expect(articles[0].topic).toBe("cats");
+          });
+      });
+      it("400: return status code 400, Invalid topic query", () => {
+        return supertest(app).get("/api/articles?topic=abc").expect(400)
+          .then((res) => {
+            const { msg } = res.body;
```

Comment:
I would suggest that this isn't an invalid topic, it just isn't found. So probably a 404 in this case.